### ISSUE TITLE
Unable to get block details for certain block number

### ISSUE THREAD

**Issue by Swornatoya** [oasisprotocol/oasis-rosetta-gateway#443]
I am unable to get the block details for various block numbers using the rosetta node. 

curl --location 'http://10.213.67.9:7332/block' \
--header 'Content-Type: application/json' \
--data '{
  "network_identifier": {
    "blockchain": "Oasis",
    "network": "b11b369e0da5bb230b220127f5e7b242d385ef8c6f54906243f30af63c815535"
  },
  "block_identifier": {
    "index": 13669000
  }
}'

response received:

{
"code": 12,
"message": "unable to get block",
"retriable": true
}

**Comment by Swornatoya** [comment]
When I tried with the above block it worked but facing issues with other blocks and here is the response from node

level=error ts=2023-06-13T05:57:24.700811336Z caller=block.go:56 module=services/block msg="Block: unable to get block" height=14052685 err="rpc error: code = Unknown desc = tendermint: block query failed: height 14052685 must be less than or equal to the current blockchain height 13772832"

**Comment by pro-wh** [comment]
the gateway seems to be behaving correctly. but what made the node start resyncing? did it lose it storage somehow?

**Comment by Swornatoya** [comment]
It was some how stopped and then when started it started syncing but it synced till block number 14252360 i.e till 8th June and did not proceed further. So we had downloaded the new git repo from https://github.com/oasisprotocol/oasis-rosetta-gateway/tree/master which has CORE_BRANCH=v22.2.8 in DockerFile where as we had this parameter earlier as CORE_BRANCH=v22.1.7 . Now we have started the node with the new git repo we downloaded. Does this solves the issue that we are facing with the Node sync.

And also could you please let me know where does the data volume gets mapped for the node

**Comment by pro-wh** [comment]
22.2.8 should be able to sync past block 14252360.

but 22.1.x should have stopped long before that block, I think at block 8048955. 

https://github.com/oasisprotocol/oasis-rosetta-gateway/blob/v2.4.0/docker/Dockerfile#L97 and I'm seeing it being mounted in /data

**Comment by Swornatoya** [comment]
Yes it is, but when I check the data folder it looks empty. I have given the proper path as well in the docker-compose file that we are using

**Comment by pro-wh** [comment]
does it even create the socket file that the gateway connects to?
